metadata.json

```diff
@@ -17,6 +17,10 @@
     {
       "name": "puppet/php",
       "version_requirement": ">= 6.0.2 < 8.0.0"
+    },
+    {
+      "name": "camptocamp/systemd",
```

Comment:
looks like we didn't use this. Does the supervisor package add the systemd file?